libsolidity/codegen/ir/Common.cpp

```diff
@@ -35,6 +35,11 @@ YulArity YulArity::fromType(FunctionType const& _functionType)
 
 string IRNames::function(FunctionDefinition const& _function)
 {
+	if (_function.isConstructor())
+		return implicitConstructor(*_function.annotation().contract);
+
+	// @TODO previously, we had to distinguish creation context and runtime context,
```

Comment:
I removed this comment in some earlier PR and it re-appeared during a merge conflict I would say :)